### PR TITLE
Remove everything related to delivery app

### PR DESCRIPTION
**Pull request by ioaNikas**
Remove everything related to delivery app

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ jobs:
           name: Post sizes of apps to zapier
           command: |
             if [ "${CIRCLE_BRANCH}" = "production" ] || [ "${CIRCLE_BRANCH}" = "master" ]; then
-              curl -X POST "${ZAPPIER_WEBHOOK}" --header "Build-number:${CIRCLE_BUILD_NUM}" --header "Branch:${CIRCLE_BRANCH}" --header  "Date:`date "+%D"`" --header  Delivery-size:`cat ./dist/apps/delivery/main**.js | wc -c` --header  Marketplace-size:`cat ./dist/apps/catalog/main**.js  | wc -c` --header  Functions-size:`cat ./dist/apps/backend-functions/main**.js | wc -c`;
+              curl -X POST "${ZAPPIER_WEBHOOK}" --header "Build-number:${CIRCLE_BUILD_NUM}" --header "Branch:${CIRCLE_BRANCH}" --header  "Date:`date "+%D"`" --header  Marketplace-size:`cat ./dist/apps/catalog/main**.js  | wc -c` --header  Functions-size:`cat ./dist/apps/backend-functions/main**.js | wc -c`;
```
Comment (lsenta) [inline .circleci/config.yml]: should we add festival?

File: .circleci/config.yml [hunk 2]
```diff
@@ -139,19 +139,17 @@ jobs:
             # Generate default json for bundlesize
             # This is a quick hack to get bundlesize AND affected build working.
             # Maintain this script to keep checking new apps.
-            mkdir -p dist/apps/delivery \
-                     dist/apps/catalog \
+            mkdir -p dist/apps/catalog \
                      dist/apps/backend-functions
```
Comment (lsenta) [inline .circleci/config.yml]: should we add festival?

File: .circleci/config.yml [hunk 3]
```diff
@@ -139,19 +139,17 @@ jobs:
...
-            touch ./dist/apps/delivery/main.dummy.js \
-                  ./dist/apps/catalog/main.dummy.js \
+            touch ./dist/apps/catalog/main.dummy.js \
                   ./dist/apps/backend-functions/main.dummy.js \
```
Comment (lsenta) [inline .circleci/config.yml]: could you remove the last `\` ?
should we add festival?

File: .circleci/config.yml (hunk 1 above)
Comment (GrandSchtroumpf) [reply .circleci/config.yml]: Would be nice 👍

**Pull request merged**